### PR TITLE
Clarification

### PR DESCRIPTION
**Pull request by lem9**
Signed-off-by: Marc Delisle <marc@infomarc.info>

### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
@lem9 can you please rebase onto `QA_4_9` ?

**Review by MauricioFauth — APPROVED** [review]

File: doc/config.rst [hunk 1]
```diff
@@ -3048,9 +3048,9 @@ the files.
     :type: string
     :default: ``''``
 
-    The name of the directory where dumps can be saved.
+    The name of the local webserver directory where export files can be saved.
```
Comment (ibennetch) [inline doc/config.rst]: For me, in the context of a web application, "local" means my machine (running the web browser), and "remote" is the server (with PHP, etc). I agree we should improve this section of documentation, but think "local" could be misleading here.

Comment (PepeLopez) [reply doc/config.rst]: > think "local" could be misleading here.

I agree with that. I opened the request some days ago, and I struggled with the terms "local" and "remote". However, I decided to use "local" for these reasons:
Local means, where phpMyAdmin resides. If it's on my (local) machine like a laptop or desktop, it's there, but mostly phpMyAdmin is on a "remote server" installed, so THERE should the export be saved, "locally". When phpMyAdmin is sending a dump from the (remote) server to your (local) machine, it's still "sending remote".
"Local" means "the machine where phpMyAdmin is". 
Please, feel free to suggest a better term!

Comment (lem9) [reply doc/config.rst]: How about "The name of the webserver directory..." ?

Comment (lem9) [reply doc/config.rst]: @williamdes Sure I'll try to remember that when submitting the correction.

Comment (lem9) [reply doc/config.rst]: Feedback please?

Comment (mauriciofauth) [reply doc/config.rst]: > How about "The name of the webserver directory..." ?

I agree with that.

Comment (lem9) [reply doc/config.rst]: Thanks @mauriciofauth, please close this pull request.

**Comment by williamdes** [comment]
Replaced by #15459

**Pull request closed without merging**